### PR TITLE
Add blog post 1

### PR DESCRIPTION
**Pull request by robmarkcole**
@balloob let me know your thoughts

### CODE REVIEW

File: website/blog/2018-11-22-data-science-for-home-assistant-launched.md [hunk 1]
```diff
@@ -5,5 +5,15 @@ authorURL: http://twitter.com/balloob
...
+
+With experience and experimentation it is possible to create some really powerful and expressive automations, and this is one of the areas where Home Assistant excels. If you browse the Home Assistant forums, you will see that some users have put significant effort into crafting and optimising their automations. Sophisticated automations might have to meet multiple criteria before being triggered, or change their behaviour in response to input values such as set points. Clearly these automations add significant value for their users or they wouldn't have bothered to create them. But wouldn't it be great if we could reduce the time, effort and experience required to create and optimise automations? What new automations would we create if data showed their was a use or need for them? Is it possible to optimise automations for end users who don't give verbal feedback e.g. pets? Can the tools of modern data science, in the hands of creative and motivated Home Assistant users, be applied to answer these questions and more? Well to find out, we are excited to launch the Home Assistant Data Science website!
```
Comment (balloob) [inline website/blog/2018-11-22-data-science-for-home-assistant-launched.md]: "data showed their was a use" typo, there

File: website/blog/2018-11-22-data-science-for-home-assistant-launched.md [hunk 2]
```diff
@@ -5,5 +5,15 @@ authorURL: http://twitter.com/balloob
...
...ng started doing data science with your Home Assistant data. To accompany the website, we have created a brand new Hassio addon [Jupyterlab-lite](https://github.com/hassio-addons/addon-jupyterlab-lite), which allows you to run a data science IDE called Jupyterlab directly on your raspberry pi hosting Home Assistant. You do data analysis in [Jupyter notebooks](https://jupyter.org/), and your data never leaves your local machine. We have also created a python library called the [HASS-Data-Detective](https://github.com/robmarkcole/HASS-data-detective) which will discover your Home Assistant database and makes it super easy to get started investigating your Home Assistant data using modern data science tools such as [Pandas](https://pandas.pydata.org/). When you install the Jupyterlab-lite addon, all the python packages required to do data science are available, and we created a Getting Started notebook which shows you how to do some elementary analysis on your Home Assistant data. Going forward, we will be publishing more articles on the Data Science website showing how you can do more advanced analysis on your Home Assistant data, and show how this can feedback into your automations.
```
Comment (balloob) [inline website/blog/2018-11-22-data-science-for-home-assistant-launched.md]: Should we capitalize raspberry pi?

Comment (balloob) [reply website/blog/2018-11-22-data-science-for-home-assistant-launched.md]: Capitalize Python

File: website/blog/2018-11-22-data-science-for-home-assistant-launched.md [hunk 3]
```diff
@@ -5,5 +5,15 @@ authorURL: http://twitter.com/balloob
 authorFBID: 297400035
 ---
```
[the remark this answers is not in the document]
Comment (robmarkcole) [reply website/blog/2018-11-22-data-science-for-home-assistant-launched.md]: Is `authorFBID` Facebook? I'd rather leave that out

[the remark this answers is not in the document]
Comment (balloob) [reply website/blog/2018-11-22-data-science-for-home-assistant-launched.md]: yeah just drop it.

**Pull request merged**